openstack/elb/v3/pools/requests.go

```diff
@@ -101,6 +99,32 @@ type CreateOpts struct {
 	// The administrative state of the Pool. A valid value is true (UP)
 	// or false (DOWN).
 	AdminStateUp *bool `json:"admin_state_up,omitempty"`
+
+	// Whether enable port transparent transmission.
+	AnyPortEnable bool `json:"any_port_enable,omitempty"`
+
+	// Update protection status
+	ProtectionStatus string `json:"protection_status,omitempty"`
+
+	// Update protection reason
+	ProtectionReason *string `json:"protection_reason,omitempty"`
+
+	// Slow start.
+	SlowStart *SlowStart `json:"slow_start,omitempty"`
+
+	// The type of the backend server group
+	Type string `json:"type,omitempty"`
+
+	// The ID of the VPC where the backend server group works
+	VpcId string `json:"vpc_id,omitempty"`
```

Comment:
it seems that `member_deletion_protection_enable` field is missing.

Reply:
it only supported in the document, the api don't support this param